src/data/secondary-nav.ts

```diff
@@ -0,0 +1,122 @@
+                                slug: '/products/atlas/charts',
+                            },
+                            {
+                                name: 'Data API',
+                                slug: '/products/atlas/data-api',
+                            },
+                            {
+                                name: 'Multi-Cloud',
+                                slug: '/products/atlas/multi-cloud',
+                            },
+                        ],
+                    },
+                    {
+                        name: 'Compass',
+                        slug: '/products/compass',
+                        dropDownItems: [],
+                    },
+                    {
+                        name: 'Realm (Mobile)',
+                        slug: '/products/realm',
+                        dropDownItems: [
+                            {
+                                name: 'SDK',
+                                slug: '/products/realm/sdk',
+                            },
+                            {
+                                name: 'Sync',
+                                slug: '/products/realm/sync',
+                            },
+                            {
+                                name: 'Studio',
+                                slug: '/products/realm/studio',
+                            },
+                        ],
+                    },
+                    {
+                        name: 'Connectors',
+                        slug: '/products/connectors',
+                        dropDownItems: [],
+                    },
+                ],
+            },
+            {
+                name: 'Languages',
+                slug: '',
+                all: 'All Languages',
+                path: '/all-topics',
+                dropDownItems: [
+                    { name: 'Go', slug: '/languages/go' },
+                    { name: 'Java', slug: '/languages/java' },
+                    { name: 'CSharp', slug: '/languages/csharp' },
+                    { name: 'C', slug: '/languages/c' },
+                ],
+            },
+            {
+                name: 'Technologies',
+                slug: '',
+                all: 'All Technologies',
+                path: '/all-topics',
+                dropDownItems: [
+                    { name: 'GCP', slug: '/technologies/gcp' },
+                    { name: 'BI', slug: '/technologies/bi' },
+                    { name: 'AWS', slug: '/technologies/aws' },
+                ],
+            },
+            {
+                name: 'Expertise Levels',
+                slug: '',
+                all: '',
+                path: '/all-topics',
+                dropDownItems: [
+                    {
+                        name: 'Introductory',
+                        slug: '/expertise-levels/introductory',
+                    },
+                    { name: 'Expert', slug: '/expertise-levels/expert' },
```

Comment:
These pages don't exist. I'm not sure what the intention was from design, so it's worth asking Mike, but you can just direct these to `/search?expertiseLevel =Introductory` now that the search is merged.